microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/deployment/SessionScopedClaimInjectionTest.java

```diff
@@ -0,0 +1,47 @@
+package org.jboss.eap.qe.microprofile.jwt.deployment;
+
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.eap.qe.microprofile.jwt.EnableJwtSubsystemSetupTask;
+import org.jboss.eap.qe.microprofile.jwt.security.rbac.jwe.JweTestCase;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.JaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SessionScopedJaxRsEndpoint;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * MP-JWT implementations are required to generate a `javax.enterprise.inject.spi.DeploymentException` for a claim value
```

Comment:
Please use `{@code foobar}` instead of backtics.

Reply:
@honza-kasik done